components/NavigationDrawer/src/MDCBottomDrawerViewController.m

```diff
@@ -49,6 +50,10 @@ - (void)setTrackingScrollView:(UIScrollView *)trackingScrollView {
   _transitionController.trackingScrollView = trackingScrollView;
 }
 
+- (MDCBottomDrawerState)state {
```

Comment:
Naming/nit: Please rename this to be clearer that it's the drawer's state, not some state of the VC.